proxy/http/HttpConfig.h

```diff
@@ -330,6 +330,16 @@ enum {
   http_stat_count
 };
 
+enum CacheOpenWriteFailAction_t {
+  CACHE_WL_FAIL_ACTION_DEFAULT                           = 0x00,
```

Comment:
Why not just 0, 1 etc. instead of 0x00, 0x01?

Reply:
Hmm..good question. I don’t quite remember why I picked hex numbers. This PR is just moving them around, fwiw. I don’t really have a preference, but  I do have an action item to remove the collapsed forwarding plugin and clean up related configs for 10.0.0 and as part of that I can change them to simple decimals,  unless you feel strongly that I should do it now.

Reply:
No it's not a significant issue.

Reply:
Cool, closing this now, will clean it up in 10.0